requirements.txt

```diff
@@ -1,2 +1,3 @@
 networkx>=2.0
 numpy
+inflection
```

Comment:
I normally sort dependencies alphabetically both here and in `meta.yaml` file for conda recipes. That way it's easy to find a thing.